DigitalLearningSolutions.Data.Tests/Services/CourseCompletionServiceTests.cs

```diff
@@ -10,142 +11,175 @@
-
-            // Then
-            result.Should().BeEquivalentTo(new CourseCompletion(
-                customisationId,
-                "Entry Level - Win 7, Office 2010",
-                "Basic Introductory Course",
-                false,
-                DateTime.Parse("2014-08-26 15:23:58.620"),
-                DateTime.Parse("2014-08-26 15:26:03.260"),
-                0,
-                true,
-                85,
-                85,
-                100,
-                53,
-                1,
-                4,
-                10,
-                10
-            ));
+            using (new TransactionScope())
+            {
+                // Given
+                const int candidateId = 118938;
+                const int customisationId = 9117;
+                courseCompletionTestHelper.AddCertificationToCourse(customisationId);
+
+                // When
+                var result = courseCompletionService.GetCourseCompletion(candidateId, customisationId);
+
+                // Then
+                result.Should().BeEquivalentTo(new CourseCompletion(
+                    customisationId,
+                    "Entry Level - Win 7, Office 2010",
+                    "Basic Introductory Course",
+                    true,
+                    DateTime.Parse("2014-08-26 15:23:58.620"),
+                    DateTime.Parse("2014-08-26 15:26:03.260"),
+                    0,
+                    true,
+                    85,
+                    85,
+                    100,
+                    53,
+                    1,
+                    4,
+                    10,
+                    10
+                ));
+            }
         }
 
         [Test]
         public void Get_course_completion_should_return_course_completion_for_course_without_tutorials()
+        {
+            using (new TransactionScope())
+            {
+                // Given
+                const int candidateId = 1;
+                const int customisationId = 100;
+                courseCompletionTestHelper.AddCertificationToCourse(customisationId);
+
+                // When
+                var result = courseCompletionService.GetCourseCompletion(candidateId, customisationId);
+
+                // Then
+                result.Should().BeEquivalentTo(new CourseCompletion(
+                    customisationId,
+                    "Entry Level - Win XP, Office 2003/07 OLD",
+                    "Standard",
+                    true,
+                    null,
+                    null,
+                    0,
+                    true,
+                    85,
+                    85,
+                    100,
+                    29,
+                    0,
+                    0,
+                    6,
+                    10
+                ));
+            }
+        }
+
+        [Test]
+        public void Get_course_completion_should_return_null_when_course_does_not_exist()

```

Comment:
Should this name be swapped with the test below? 🙂

Reply:
Yes, sorry! Thanks for catching that, fixed in f072c1d 🙂